src/components/user/uc/users.uc.js

```diff
@@ -26,16 +28,51 @@ const getUserRelatedData = async (id) => {
 	});
 
 	const account = await accountRepo.getUserAccount(id);
-	returnArray.push({
-		scope: 'account',
-		data: account,
-	});
+	if (account) {
+		returnArray.push({
+			scope: 'account',
+			data: account,
+		});
+	}
+
+	const pseudonyms = await pseudonymUC.getPseudonymsForUser(id);
+	if (pseudonyms.length > 0) {
+		returnArray.push({
+			scope: 'pseudonym',
```

Comment:
scope: 'pseudonym' object at this place is reverted by newer changes from @alriese @Metauriel.

Reply:
the pseudonym uc should already return the pseudonym object in the correct format.